tests/subscriptions/entity_subscriptions/test_entity_subscriptions.py

```diff
@@ -124,35 +130,38 @@
             ),
         ],
         True,
+        1,
         id="Metrics sets subscription of type SNQL",
     ),
 ]
 
 
-@pytest.mark.parametrize("entity_subscription, creation_dict, exception", TESTS)
+@pytest.mark.parametrize("entity_subscription, organization, exception", TESTS)
 def test_basic(
     entity_subscription: Type[EntitySubscription],
-    creation_dict: Mapping[str, Any],
+    organization: Optional[int],
     exception: Optional[Type[Exception]],
 ) -> None:
+    es = entity_subscription()
     if exception is not None:
         with pytest.raises(exception):
-            entity_subscription(**creation_dict)
-    else:
-        es = entity_subscription(**creation_dict)
-        assert es.to_dict() == creation_dict["data_dict"]
+            es.get_entity_subscription_conditions_for_snql(None, organization)
```

Comment:
I don't think you changed this test correctly. You shouldn't remove the `else` clause, that will skip a bunch of the tests. It should look like:

```
if exception is not None:
    with pytest.raises(exception):
        something()
else:
    something()
```

Also, you should assert something on this call in the successful case. As written this doesn't actually verify the conditions are correct in any way.